src/seedu/addressbook/data/person/Address.java

```diff
@@ -56,4 +57,10 @@ public int hashCode() {
     public boolean isPrivate() {
         return isPrivate;
     }
+    
+	@Override
+	public String getPrintableString(Printable... printables) {
```

Comment:
No need to keep mentioning `Printable... printables`. Recollect from the discussion.